cli/enrol.yaml

```diff
@@ -0,0 +1,72 @@
+## Every recipe file must provide at least the component property.
+## Based on the plugin type plugin specific files files will be generated.
+component: enrol_plugintest
+
+## Human readable name of the plugin.
+name: Enrol plugin
+
+## Human readable release number.
+release: "0.1.0"
+
+## Plugin version number, e.g. 2016062100. Will be set to current date if left empty.
+#version: 2016121200
+
+## Required Moodle version, e.g. 2015051100 or "2.9".
+requires: "2.9"
+
+## Plugin maturity level. Possible options are MATURIY_ALPHA, MATURITY_BETA,
+## MATURITY_RC or MATURIY_STABLE.
+maturity: MATURITY_BETA
+
+## Copyright holder(s) of the generated files and classes.
+copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
+
+## Features flags can control generation of optional files/code fragments.
+features:
+    readme: true
+    license: true
+    install: true
+    settings: true
+
+## Allows manual enrolment. Requires enrol/<enrolname>:enrol capability.
+allow_enrol: 'false'
```

Comment:
Like I commented in the other pull request (for auth plugins), these plugin type specific flags should be probably grouped somehow. What about something like this:
```
features:
    readme: true
    # Here would be general feature flags common for all plugin types
enrol_features:
    allow_enrol: false
    allow_unenrol: true
    # Here would be enrol-specific features
auth_features:
    # Here would be auth-specific features
```